ratelimiter/src/main/java/com/flipkart/varadhi/qos/RateLimiter.java

```diff
@@ -0,0 +1,7 @@
+package com.flipkart.varadhi.qos;
+
+public interface RateLimiter<R, L> {
+    // assumes 1 QPS
+    // for batch request (to be considered as multi QPS), consider creating a new API
```

Comment:
this comment looks incorrect.

R & L probably does not come with any assumptions. Why do you say 1 QPS?

Reply:
missed removing this. Its generic and doesn't assumes anything now